### DISCUSSION TITLE
Entry_with_update.py - TypeError: unsupported operand type(s) for |: 'dict' and 'dict'

### DISCUSSION THREAD

**Discussion by djaiswal21** [lllyasviel/Fooocus#2668] [Q&A] [answered]
I ran all the commands and was stuck at this one. 
I ran the `conda activate fooocus`
then did `python entry_with_update.py`

I get the error : 

![image](https://github.com/lllyasviel/Fooocus/assets/31632896/3eb292c2-61c2-44cc-a1cd-ec6412ca85e6)

has anyone lese faced this issue ?

**Comment by mashb1t** [comment, marked as answer]
Fooocus is only compatible with Python 3.10 or higher, not 3.8

**Reply by djaiswal21** [reply]
Yes Once I updated the Python it works ! thank you
